common/src/main/java/com/microsoft/identity/common/nativeauth/internal/controllers/NativeAuthMsalController.kt

```diff
@@ -365,15 +452,13 @@ class NativeAuthMsalController : BaseNativeAuthController() {
                         correlationId = result.correlationId
                     )
                 }
-                is SignInChallengeApiResult.IntrospectRequired -> {
-                    TODO()
-                }
-                is SignInChallengeApiResult.UnknownError -> {
+                is SignInChallengeApiResult.IntrospectRequired, is SignInChallengeApiResult.UnknownError -> {
```

Comment:
Just double checking - we group this with the unknown error since we should never get an introspect required error when using the default challenge method?

Reply:
Yes. If the API returns introspect_required when the default challenge is called, the SDK should never get into a state where resendCode() can be called. Hence, if the SDK is in a state where resend code can be called, there should be a default auth method that was queried in the first place, hence introspect_required is an unexpected response.